generators/bootstrap-application-server/generator.ts

```diff
@@ -209,17 +205,7 @@ export default class BoostrapApplicationServer extends BaseApplicationGenerator
   }
 
   get postPreparingEachEntity() {
-    return this.asPostPreparingEachEntityTaskGroup({
-      processDerivedPrimaryKeyFields({ entity }) {
-        const primaryKey = entity.primaryKey;
-        if (!primaryKey || primaryKey.composite || !primaryKey.derived) {
-          return;
-        }
-        // derivedPrimary uses '@MapsId', which requires for each relationship id field to have corresponding field in the model
-        const derivedFields = primaryKey.derivedFields;
-        entity.fields.unshift(...derivedFields);
-      },
-    });
+    return this.asPostPreparingEachEntityTaskGroup;
```

Comment:
Leftover move derived primary key back here

Reply:
duplicated the code and deprecated the server version

Reply:
Done as asked